plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -222,6 +222,9 @@ public DistributedQueryRunner build()
                     hiveProperties.put("hive.metastore", "file");
                     hiveProperties.put("hive.metastore.catalog.dir", queryRunner.getCoordinator().getBaseDataDir().resolve("hive_data").toString());
                 }
+                if (!hiveProperties.buildOrThrow().containsKey("fs.hadoop.enabled")) {
```

Comment:
Should we not use the new navy file systems for most of the tests?

Reply:
All those tests were already migrated to native-fs, by explicitly setting fs.hadoop.enabled to false, and enabling one of s3/azure/gcs implementations

Reply:
cool

Reply:
So why do we need to add this?

Reply:
@electrum there are still tests with no `fs.hadoop.enabled` values set.  When default was changed to false, tests fail with errors like `No factory set for location /tmp..` . Hence this change

Reply:
Should we change those tests to make sure they do have the fs.hadoop.enabled set to on instead

Reply:
> Should we change those tests to make sure they do have the fs.hadoop.enabled set to on instead

We could, but there will be a lot. I followed the pattern in the runner where we default to `file` metastore when no metastore is set.

Reply:
> When default was changed to false, tests fail with errors like No factory set for location /tmp..`


Here is a  stacktrace

<details>
  <summary>stacktrace</summary>
  io.trino.spi.TrinoException: Could not read database schema

	at io.trino.plugin.hive.metastore.file.FileHiveMetastore.readFile(FileHiveMetastore.java:1406)
	at io.trino.plugin.hive.metastore.file.FileHiveMetastore.readSchemaFile(FileHiveMetastore.java:1391)
	at io.trino.plugin.hive.metastore.file.FileHiveMetastore.getDatabase(FileHiveMetastore.java:285)
	at io.trino.plugin.hive.metastore.file.FileHiveMetastore.createDatabase(FileHiveMetastore.java:198)
	at io.trino.metastore.tracing.TracingHiveMetastore.lambda$createDatabase$9(TracingHiveMetastore.java:173)
	at io.trino.metastore.tracing.Tracing.lambda$withTracing$0(Tracing.java:31)
	at io.trino.metastore.tracing.Tracing.withTracing(Tracing.java:39)
	at io.trino.metastore.tracing.Tracing.withTracing(Tracing.java:30)
	at io.trino.metastore.tracing.TracingHiveMetastore.createDatabase(TracingHiveMetastore.java:173)
	at io.trino.plugin.hive.metastore.cache.CachingHiveMetastore.createDatabase(CachingHiveMetastore.java:572)
	at io.trino.plugin.deltalake.TestDeltaLakeProjectionPushdownPlans.createPlanTester(TestDeltaLakeProjectionPushdownPlans.java:119)
</details>


It looks to me that we need to have a mapping for `io.trino.filesystem.local.LocalFileSystemFactory` in `FileSystemModule` or an extension of `FileSystemModule` that we are using for testing purposes in order to cope with local fs locations.

Reply:
Supporting local fs would be cool anyway